src/main/java/com/topicus/CFPApplication/persistence/RequestCategorizedDraftsService.java

```diff
@@ -22,11 +22,11 @@ public RequestCategorizedDraftsService(PresentationDraftRepository presentationD
 	}
 	
 	//Method gets called when both Conference and Category exist
-//	public Iterable<PresentationDraft> findPresentationDraftsByCategory(Conference conference, String category){
-//		Iterable<PresentationDraft> allDraftsFromConference = this.presentationDraftRepository.findPresentationDraftByConferenceId(conference.getId());
+//	public Iterable<PresentationDraftRequest> findPresentationDraftsByCategory(Conference conference, String category){
+//		Iterable<PresentationDraftRequest> allDraftsFromConference = this.presentationDraftRepository.findPresentationDraftByConferenceId(conference.getId());
 //		presentationDraftConference = new PresentationDraftConference();
 //
```

Comment:
comments

Reply:
Deze heb ik niet gemaakt en heb geen idee waar die voor is. Ik laat hem staan en heb een ticket aangemaakt om die uiteindelijk op te ruimen als we die niet nodig hebben.